main.go

```diff
@@ -52,19 +68,21 @@ func main() {
 	if args.configFile != "" {
 		viper.SetConfigFile(args.configFile)
 	}
-	err := viper.ReadInConfig()
+
+	err = viper.ReadInConfig()
 	if err != nil { // Handle errors reading the config file
-		panic(fmt.Errorf("fatal error config file: %s", err))
+		klog.Fatalf("fatal error config file: %s", err)
+		os.Exit(1)
```

Comment:
@chentex `Fatalf` exists with `os.Exit(255)` after logging. (see [here](https://pkg.go.dev/k8s.io/klog@v0.3.2#Fatalf))

No need for this ^^

Reply:
Thank you @nimrodshn I am still determining how to continue with this, because we always get a mix of log formats in the output and I don't like that very much, pretty sure this will change.

Reply:
> While it gives a cleaner output, once you go debug mode (level 2+) it starts mixing them.

@chentex Hmm how often do we use `level 2+` (i.e. `Debug+` ?) in our code ? I'm hesitant about this move.. we should ideally try and be consistent with our loggers, can we adopt the `glog` fork `ocm-sdk` uses?

Reply:
I will take a look on implementing it today, I guess this is going to be the cleanest way to do it.